tests/tensor/rewriting/test_math.py

```diff
@@ -2031,6 +2031,56 @@ def test_exp_log_nested(self, nested_expression, expected_switches):
         assert len(ops_graph) == expected_switches
 
 
+class TestSqrSqrt:
+    def setup_method(self):
+        mode = get_default_mode()
+        self.mode = mode.including(
+            "local_sqrt_sqr",
+        ).excluding("fusion")
+        self.rng = np.random.default_rng()
+
+    def test_sqr_sqrt(self):
+        # sqrt(x) ** 2 -> x
+        data = self.rng.normal(size=(4, 3)).astype(config.floatX)
+        x = pt.tensor("x", shape=(None, None))
+        f = function([x], sqr(sqrt(x)), mode=self.mode)
```

Comment:
For this simple rewrite I would just call rewrite_graph and use `assert_equal_computations` with the expected graph. It's faster and more explicit.

No need to compile a function and evaluate when the expected graph is obviously correct